app/client/components/delivery/records/deliveryAddressStep.tsx

```diff
@@ -0,0 +1,582 @@
+                background-color: transparent;
+                :hover {
+                  background-color: transparent;
+                }
+              `}
+            >
+              Discard changes
+            </Button>
+          </div>
+        </form>
+        {productsAffected.length > 1 && (
+          <p
+            css={css`
+              ${textSans.medium()};
+              background-color: ${palette.neutral[97]};
+              padding: ${space[5]}px ${space[5]}px ${space[5]}px 49px;
+              margin: ${space[5]}px 0 ${space[3]}px;
+              position: relative;
+            `}
+          >
+            <i
+              css={css`
+                width: 17px;
+                height: 17px;
+                position: absolute;
+                top: ${space[5]}px;
+                left: ${space[5]}px;
+              `}
+            >
+              <InfoIconDark fillColor={palette.brand[500]} />
+            </i>
+            If you need seperate delivery addresses for each of your
+            subscriptions, please{" "}
+            <span
+              css={css`
+                cursor: pointer;
+                color: ${palette.brand[500]};
+                text-decoration: underline;
+              `}
+              onClick={() =>
+                setCallCentreNumbersVisibility(!showCallCentreNumbers)
+              }
+            >
+              contact us
+            </span>
+            .
+          </p>
+        )}
+        {showCallCentreNumbers && <CallCentreEmailAndNumbers />}
+      </>
+    );
+  };
+
+  const renderConfirmation = () => (
+    <>
+      <div
+        css={css`
+          padding: ${space[3]}px;
+          ${minWidth.tablet} {
+            padding: ${space[5]}px;
+          }
+        `}
+      >
+        <SuccessMessage
+          additionalCss={css`
+            margin-bottom: 0;
+          `}
+          message={`We have successfully updated your delivery details for your subscription${deliveryAddressContext.productsAffected &&
+            deliveryAddressContext.productsAffected.length > 1 &&
+            "s"}. You will shortly receive a confirmation email.`}
+        />
+      </div>
+      <ReadOnlyAddressDisplay
+        address={newAddress}
+        instructions={
+          (props.enableDeliveryInstructions &&
+            deliveryAddressContext.address?.instructions) ||
+          undefined
+        }
+      />
+    </>
+  );
+
+  if (
+    status === Status.EDIT ||
+    status === Status.PENDING ||
+    status === Status.VALIDATION_ERROR
+  ) {
+    return (
+      <div
+        css={css`
+          padding: ${space[3]}px;
+          ${minWidth.tablet} {
+            padding: ${space[5]}px;
+          }
+        `}
+      >
+        <MembersDatApiAsyncLoader
+          render={renderDeliveryAddressForm}
+          fetch={createProductDetailFetcher(ProductTypes.contentSubscriptions)}
+          loadingMessage={"Loading delivery details..."}
+        />
```

Comment:
as discussed IRL we'll do a follow-up PR to avoid having to make this additional call (off the back of https://github.com/guardian/manage-frontend/pull/385)